packages/slate-react/src/utils/scroll-to-selection.js

```diff
@@ -38,7 +38,9 @@ function findScrollContainer(el) {
     parent = parent.parentNode
   }
 
-  if (!scroller) return window
+  if (!scroller) {
```

Comment:
Can you add a `COMPAT:` comment here for future reference?

Reply:
Added. Does that seem appropriate to you?